pkg/proxy/config/config.go

```diff
@@ -31,28 +31,21 @@ type Operation int
 
 // These are the available operation types.
 const (
-	SET Operation = iota
-	ADD
+	ADD Operation = iota
```

Comment:
OK - so I will explicitly change it to contain: "ADD", "UPDATE" and "REMOVE" actions, and will just handle ADD and UPDATE the same in the handling code - I agree this is cleaner and will be consistent with our codebase.